src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -1060,15 +1050,14 @@ def readCondorLog(self, job):
             cres = condor.read_events(logfileobj, 1)
             ulog = list(cres)
             if len(ulog) > 0:
-                if all(key in ulog[-1] for key in ("TriggerEventTypeNumber", "QDate", "JobStartDate",
-                                                   "EnteredCurrentStatus", "MATCH_EXP_JOBGLIDEIN_CMSSite",
-                                                   "WMAgent_JobID")):
+                if all(key in ulog[-1] for key in ("TriggerEventTypeNumber", "QDate", "EnteredCurrentStatus",
```

Comment:
That's fine too.  Just the assumption that _all_ data was present (particularly, `JOBGLIDEIN_CMSSite` is only relevant if the job was running - might not be true for held or idle jobs) was causing the code to not work.

Having slightly wrong data may be the best we can do, unfortunately.